.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+### Description
+[PLEASE REMOVE] Describe the goal of this PR and any add any relevant context.
+
+### Test plan
+[PLEASE REMOVE] How did you test this PR? Please write down any manual commands you used and note down tests that you have written.
+
+### Issue(s) fixed
+[PLEASE REMOVE] List all issues, separate by returns, that this PR addresses.
+
+### Changelog
+[PLEASE REMOVE] If this PR introduces a fix or feature that should be the upcoming release notes, please add the following line below - `+changelog: <description of feature>`
```

Comment:
Is this `+changelog` syntax a common convention?

Reply:
I've seen it in a couple places, not sure if it's a convention or needs to be. This just lets us regex et's pr history for all release notes.

Reply:
Where have you seen it? That's helpful context for the PR summary. My main thought here is: the template is a good idea, but we should follow existing conventions as much as possible, and point to sources of those conventions.

Reply:
Hmm, makes sense to keep development across torch repos consistent. PyTorch makes you add a `release notes: <subject>` label, which should let you [automatically generate your release notes](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes) by regexing for `release notes*` labels. We can do that as well.